mlflow/store/tracking/abstract_store.py

```diff
@@ -294,12 +295,64 @@ def delete_traces(
             experiment_id: ID of the associated experiment.
             max_timestamp_millis: The maximum timestamp in milliseconds since the UNIX epoch for
                 deleting traces. Traces older than or equal to this timestamp will be deleted.
-            max_traces: The maximum number of traces to delete.
+            max_traces: The maximum number of traces to delete. If max_traces is specified, and
+                it is less than the number of traces that would be deleted based on the
+                max_timestamp_millis, the oldest traces will be deleted first.
             request_ids: A set of request IDs to delete.
 
+            Example usage:
+
+                .. code-block:: python
```

Comment:
```suggestion
                .. code-block:: python
                    :test:
```

can we test this example?